docs/tutorial.md

```diff
@@ -394,16 +462,26 @@ conflicts.
 Let's try one final command for changing the contents of an exiting commit. That
 command is `jj diffedit`, which lets you edit the contents of a commit without
 checking it out.
+
 ```shell
 $ jj diffedit -r @-
-Using default editor 'meld'; you can change this by setting ui.diff-editor
-Created 70985eaa924f ABC
+Using default editor ':builtin'; you can change this by setting ui.diff-editor
+Created msxluwsl af7cbc92 (empty) ABC
 Rebased 1 descendant commits
-Working copy now at: 1c72cd50525d ABCD
+New conflicts appeared in these commits:
+  szwotywq 280ab832 (conflict) ABCD
+To resolve the conflicts, start by updating to it:
+  jj new szwotywqmyqv
+Then use `jj resolve`, or edit the conflict markers in the file directly.
+Once the conflicts are resolved, you may want inspect the result with `jj diff`.
+Then run `jj squash` to move the resolution into the conflicted commit.
+Working copy now at: szwotywq 280ab832 (conflict) ABCD
+Parent commit      : msxluwsl af7cbc92 (empty) ABC
 Added 0 files, modified 1 files, removed 0 files
 ```
-When Meld starts, edit the right side by e.g. adding something to the first
-line. Then save the changes and close Meld. You can now inspect the rewritten
+
+In the diff editor, edit the right side by e.g. adding something to the first
+line. Then save the changes and close it. You can now inspect the rewritten
```

Comment:
Maybe "Then save the changes and close it" => "Press 'c' to save the changes and close it"